fluentd/README.md

```diff
@@ -7,3 +7,16 @@ Following are the environment variables that can be modified to adjust the confi
 | Environment Variable | Description |Example|
 |----------------------|-------------|---|
 | `OCP_OPERATIONS_PROJECTS`| The list of project or patterns for which messages will be sent to the operations indices|`OCP_OPERATIONS_PROJECTS="default openshift openshift-"`
+
+## Cri-o Formatted Container Logs
+In order to enable cri-o logs parsing, it is necessary to mount 
+`node-config.yaml` from the host inside the fluentd container to this path:
```

Comment:
Why not have a run.sh script for fluentd determine that via the log file contents itself?

Reply:
What if there are no log files yet?

Reply:
Good point.

But this discussion about multiple container runtimes is making this more of a question of, "how can we detect what log format to use?"

If we configure two `in_tail` plugins, which assume that the container runtimes keep their log files in separate directory hierarchies which path prefixes unique to each container runtime, then those plugins would work without having to detect the file contents.